### ISSUE TITLE
error while loading shared libraries

### ISSUE THREAD

**Issue by ASB1993** [tensordot/syntaxdot#201]
Hi, 
so I tried to prepare the label files as described in https://github.com/tensordot/syntaxdot/blob/main/doc/finetune.md with the command `syntaxdot prepare /mnt/c/Users/bartl/Downloads/syntaxdot.conf /mnt/c/Users/bartl/Downloads/train.txt`. However, it throws an error, namely: "syntaxdot: error while loading shared libraries: libtorch_cpu.so: cannot open shared object file: No such file or directory". I checked whether pytorch was installed correctly by running `python -c "import torch; print(torch.eye(3))"` and it gives me output.
What could be wrong here?
